.github/wiki/Contribution.md

```diff
@@ -0,0 +1,242 @@
+# Adding a new addon
+
+There are 2 types of addon, Mix addon and Phoenix addon.
+
+Phoenix addon consists of 3 types.
+1. API
+2. Live
+3. Web
+
+[[https://user-images.githubusercontent.com/11751745/157225788-9f8c28ad-058b-4994-851e-82c98854ddcd.png|alt=structure]]
```

Comment:
Hi Top, I think we need to upload this file to the Wiki folder 😄 

![image](https://user-images.githubusercontent.com/11751745/157361268-4303be58-e808-4bac-ad66-2244208670ea.png)

And then we can refer that image by `![diagram](assets/images/...)`

Reply:
Fixed in ab169c9